discovery/eureka_test.go

```diff
@@ -0,0 +1,119 @@
+package discovery
+
+import (
+	"gopkg.in/ory-am/dockertest.v3"
+	dc "github.com/fsouza/go-dockerclient"
+	"github.com/jaume-pinyol/fargo"
+	"log"
+	"os"
+	"testing"
+	"strconv"
+	"github.com/op/go-logging"
+)
+
+var eurekaTestPort int = 8080
+var eurekaTestUrl string = "http://127.0.0.1:" + strconv.Itoa(eurekaTestPort) + "/eureka"
+
+func TestMain(m *testing.M) {
+	// uses a sensible default on windows (tcp/http) and linux/osx (socket)
+	pool, err := dockertest.NewPool("")
+	if err != nil {
+		log.Fatalf("Could not connect to docker: %s", err)
+	}
+	// pulls an image, creates a container based on it and runs it
+	resource, err := pool.RunWithOptions(&dockertest.RunOptions{
```

Comment:
this resource is the Eureka container running, isn't it? a more descriptive name could be great 😄

Reply:
:+1: . I was following the examples to strictly :smile_cat: